airflow/dags/operations/pre_sync_documents_to_kernel_operations.py

```diff
@@ -26,15 +26,27 @@ def get_sps_packages(scilista_file_path, xc_dir_name, proc_dir_name):
     sps_packages_list = []
 
     with open(scilista_file_path) as scilista:
-        for acron_issue in scilista.readlines():
-            # Verifica se comando DEL está indicado no fascículo
-            if not acron_issue.strip().lower().endswith("del"):
-                filename_pattern = "*{}.zip".format("_".join(acron_issue.split()))
-                Logger.info("Reading ZIP files pattern: %s", filename_pattern)
-                for source in sorted(xc_dir_path.glob(filename_pattern)):
+        for scilista_item in scilista.readlines():
+            acron_issue = scilista_item.strip().split()
+            if len(acron_issue) != 2:
+                continue
+            filename_pattern = "*{}.zip".format("_".join(acron_issue))
+            Logger.info("Reading ZIP files pattern: %s", filename_pattern)
+            for source in sorted(xc_dir_path.glob(filename_pattern)):
+                sps_packages_list.append(str(proc_dir_path / source.name))
+                if os.path.isfile(str(proc_dir_path / source.name)):
```

Comment:
@robertatakenaka Montei uma scilista com alguns fascículos de pacotes que já estavam lá no diretório de destino. A aplicação não tenta copiá-los, o que está certo. Só que este `continue` está fazendo com que caia no `else` do `for`, logando o padrão como se nenhum arquivo tivesse sido encontrado.

Reply:
@patymori o problema não é no `continue`, mas sim porque já não existem mais os pacotes no _source_, então ele entra no `else` do `for`. Vou ter que alterar todo o código.

Reply:
@patymori usar `for` com generator, sempre no final entra em `else`!!! e por isso, está resultando em um comportamento não esperado